netket/optimizer/jax/_sr_onthefly.py

```diff
@@ -0,0 +1,92 @@
+import jax
+import jax.numpy as jnp
+from functools import partial
+
+
+# onthefly SR
+# since we cant store O if #samples x #params is too large
+
+
+def O_jvp(x, theta, v, vlogwf):
```

Comment:
can you rename `vlogwf` to be consistent with what we use in the rest of netket? I'd say `forward_fn` should be good, or something else if you prefer.

Reply:
also below

Reply:
d3f6780a0eea376620c45af90ccd6d0590a4c851 done.